modules.d/90kernel-modules/module-setup.sh

```diff
@@ -133,7 +133,6 @@ installkernel() {
         [[ $arch == aarch64 ]] && arch=arm64
         instmods "=crypto" "=arch/$arch/crypto" "=drivers/crypto"
     fi
-    :
```

Comment:
commit c9e5b75308d858dfb865d3db0a7bad1d6062348d
Author: Harald Hoyer <harald@redhat.com>
Date:   Mon Mar 18 09:29:08 2013 +0100

    kernel-modules/module-setup.sh: don't fail hard on a missing module

Reply:
Ah, I got it. Thanks, I won't touch this line then.